.github/workflows/linter.yml

```
@@ -0,0 +1,26 @@
+---
+name: Lint Code Base
+on:
+  push:
+  pull_request:
+    types:
+      - opened
+    branches:
+      - master
+jobs:
+  build:
+    name: Lint Code Base
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Lint Code Base
+        uses: github/super-linter@v4
+        env:
+          VALIDATE_PHP_PSALM: false
```

```suggestion
          LINTER_RULES_PATH: ./
          VALIDATE_PHP_PSALM: false
```
@Faggioni You may need to set the linter rules path so that super linter reads linter configurations from the root folder

---

Hey @kodnificent, by default, that property is set to the .github/linters directory.

Currently, I'm not implementing any linters and probably will fail the check-ups, but I would like to set the issues template on the repo.

---

Okay, that's fine